test/integration/data.py

```diff
@@ -143,6 +145,9 @@
 
 ES_DATASET_BASE_URL = "https://raw.githubusercontent.com/elastic/elasticsearch/6857d305270be3d987689fda37cc84b7bc18fbb3/x-pack/plugin/sql/qa/src/main/resources/"
 
+KIBANA_SAMPLES_BASE_URL = "https://raw.githubusercontent.com/elastic/kibana/master/src/legacy/server/sample_data/data_sets"
```

Comment:
Here you're using the `master` branch whereas for the ES data set above you used a specific commit.

Have you considered using a specific commit for Kibana too?  If you use `master` they can break your tests by changing the data and/or directory structure, forcing you to react immediately. Whereas if you use a specific commit you can just update it once per year at a time of your choosing.

Reply:
Thanks, I've changed it to stay consistent.
(My late thought was to be "notified" quickly on data set change, but otoh it does seem that Kibana's folder structure might change.)